app/views/application/_tasks_table.html.erb

```diff
@@ -13,7 +13,11 @@
   <tbody>
     <% tasks.each do |task| %>
     <tr>
-      <td><%= task.user_name %></td>
+      <% if task.user %>
+        <td>Assigned to <%= task.user_name %></td>
+      <% else %>
+        <td>No user</td>
```

Comment:
In the drop down when creating a task the text is `Unassigned` should use the same word here.